tests/shared/src/test/scala/cats/tests/CatsSuite.scala

```diff
@@ -16,13 +16,15 @@ import scala.util.{Failure, Success, Try}
  * An opinionated stack of traits to improve consistency and reduce
  * boilerplate in Cats tests.
  */
-trait CatsSuite extends FunSuite with Matchers with Discipline with AllInstances with AllSyntax with TestInstances {
+trait CatsSuite extends FunSuite with Matchers with Discipline with AllInstances with AllSyntax with TestInstances with StrictCatsEquality {
   implicit override val generatorDrivenConfig: PropertyCheckConfiguration =
     PropertyCheckConfiguration(
       minSuccessful = Platform.minSuccessful,
       maxDiscardedFactor = Platform.maxDiscardedFactor)
-  // disable scalatest's ===
-  override def convertToEqualizer[T](left: T): Equalizer[T] = ???
+
+  // disable Eq syntax (by making `eqSyntax` not implicit), since it collides
+  // with scalactic's equality
+  override def eqSyntax[A: Eq](a: A): EqOps[A] = new EqOps[A](a)
```

Comment:
As per previous comment, consider moving this into its own trait in `main`, perhaps in conjunction with the other traits. 
This would also mean that users need not import `cats.syntax.EqOps`